tutorials/toolkits/axes_grid.py

```diff
@@ -1,115 +1,84 @@
 r"""
-==============================
-Overview of axes_grid1 toolkit
-==============================
-
-Controlling the layout of plots with the
-:mod:`mpl_toolkits.axes_grid1` toolkit.
-
 .. _axes_grid1_users-guide-index:
 
+==========================================
+Overview of :mod:`mpl_toolkits.axes_grid1`
+==========================================
 
-What is axes_grid1 toolkit?
-===========================
+:mod:`.axes_grid1` provides the following features:
 
-:mod:`mpl_toolkits.axes_grid1` is a collection of helper classes to ease
-displaying (multiple) images with matplotlib. In matplotlib, the axes location
-(and size) is specified in the normalized figure coordinates, which
-may not be ideal for displaying images that needs to have a given
-aspect ratio.  For example, it helps if you have a colorbar whose
-height always matches that of the image.  `ImageGrid`_, `RGB Axes`_ and
-`AxesDivider`_ are helper classes that deal with adjusting the
-location of (multiple) Axes.  They provides a framework to adjust the
-position of multiple axes at the drawing time. `ParasiteAxes`_
-provides twinx(or twiny)-like features so that you can plot different
-data (e.g., different y-scale) in a same Axes. `AnchoredArtists`_
-includes custom artists which are placed at some anchored position,
-like the legend.
+- Helper classes (ImageGrid_, RGBAxes_, AxesDivider_) to ease the layout of
+  axes displaying images with a fixed aspect ratio while satisfying additional
+  constraints (matching the heights of a colorbar and an image, or fixing the
+  padding between images);
+- ParasiteAxes_ (twinx/twiny-like features so that you can plot different data
+  (e.g., different y-scale) in a same Axes);
+- AnchoredArtists_ (custom artists which are placed at an anchored position,
+  similarly to legends).
 
 .. figure:: ../../gallery/axes_grid1/images/sphx_glr_demo_axes_grid_001.png
    :target: ../../gallery/axes_grid1/demo_axes_grid.html
    :align: center
    :scale: 50
 
-   Demo Axes Grid
-
-
 axes_grid1
 ==========
 
 ImageGrid
 ---------
 
-A grid of Axes.
-
-In Matplotlib, the axes location (and size) is specified in normalized
-figure coordinates. This may not be ideal for images that needs to be
-displayed with a given aspect ratio; for example, it is difficult to
-display multiple images of a same size with some fixed padding between
-them.  `~.axes_grid1.axes_grid.ImageGrid` can be used in such a case; see its
-docs for a detailed list of the parameters it accepts.
+In Matplotlib, axes location and size are normally specified in normalized
```

Comment:
Drop doubled 'normal'
```suggestion
In Matplotlib, axes location and size are usualy specified in normalized
```